proxies/nodejs/handlers/user.ts

```diff
@@ -1,26 +0,0 @@
-import Koa from 'koa'
```

Comment:
if you're deleting this theres also a c# equivalent still isn't there?

Reply:
theres not

Reply:
but there is a HelloWorld controller that I deleted